modules/gumgumBidAdapter.js

```diff
@@ -294,6 +294,7 @@ function buildRequests(validBidRequests, bidderRequest) {
   const gdprConsent = bidderRequest && bidderRequest.gdprConsent;
   const uspConsent = bidderRequest && bidderRequest.uspConsent;
   const timeout = config.getConfig('bidderTimeout');
+  const coppa = config.getConfig('coppa') === true ? 1 : 0;
```

Comment:
fwiw the line above this is the wrong way to get timeout; the timeout from requestBids can be overridden in https://docs.prebid.org/dev-docs/publisher-api-reference/requestBids.html

For example, I think CafeMedia always overrides.

Reply:
@patmmccann thanks for the review and for catching this issue. I'll make a ticket to review if we're using it or not and make the necessary corrections based on those findings.